### PR TITLE
IRV - When clicking on the map, show zonal information in a more structured way

### PR DESCRIPTION
**Pull request by ptormene**
Partially addressing https://github.com/gem/oq-platform/issues/583

With this change, instead of just displaying the raw and unsorted data, the application searches the extended names and types of variables by scanning the project definition. Then the variables are presented in a structured way, collected by type.

Tests are green: https://ci.openquake.org/job/zdevel_oq-platform/252/

### CODE REVIEW

File: openquakeplatform/openquakeplatform/irv/static/irv/js/irv_viewer.js [hunk 1]
```diff
@@ -1019,22 +1028,100 @@ function mapboxGlLayerCreation() {
...
+            if (typeof features[0] !== 'undefined' && typeof features[0].properties !== 'undefined') {
+                for (var field in features[0].properties) {
+                    var nameAndType = findNameAndTypeInProjDef(field, sessionProjectDef);
+                    if ('name' in nameAndType) {
+                        isDataAvailable = true;
+                        var nameAndValue = {
+                            'name': nameAndType.name,
+                            'value': features[0].properties[field]
+                        };
+                        switch (nameAndType.type) {
```
Comment (nastasi-oq) [inline openquakeplatform/openquakeplatform/irv/static/irv/js/irv_viewer.js]: 'default:' case missing, is it correct ?

**Comment by nastasi-oq** [comment]
No tests on CI for this branch ?

File: openquakeplatform/openquakeplatform/irv/static/irv/js/irv_viewer.js (hunk 1 above)
Comment (ptormene) [inline openquakeplatform/openquakeplatform/irv/static/irv/js/irv_viewer.js]: All cases should be covered, unless the project definition is wrong. I am adding a default, giving warning in such corner case, through an alert.

**Comment by nastasi-oq** [comment]
LGTM

**Pull request merged**